neutrino.go

```diff
@@ -945,6 +953,22 @@ func NewChainService(cfg Config) (*ChainService, error) {
 		permanentPeers = cfg.AddPeers
 	}
 
+	// Adding rest peers to chainservice if spesified in config
+
+	restPeers := cfg.RestPeers
```

Comment:
This variable is really needed? (cfg.RestPeers can be directly used in the loop)